modules/core/src/test/java/org/apache/ignite/internal/metric/SystemViewSelfTest.java

```diff
@@ -909,6 +945,648 @@ private void checkScanQueryView(IgniteEx client1, IgniteEx client2, SystemView<S
         assertTrue(found1 && found2);
     }
 
+
+    /** */
+    @Test
+    public void testAtomicSequence() throws Exception {
+        try (IgniteEx g0 = startGrid(0);
+             IgniteEx g1 = startGrid(1)) {
+            IgniteAtomicSequence s1 = g0.atomicSequence("seq-1", 42, true);
+            IgniteAtomicSequence s2 = g0.atomicSequence("seq-2",
+                new AtomicConfiguration().setBackups(1).setGroupName("my-group"), 43, true);
+
+            s1.batchSize(42);
+
+            SystemView<AtomicSequenceView> seqs0 = g0.context().systemView().view(SEQUENCES_VIEW);
+            SystemView<AtomicSequenceView> seqs1 = g1.context().systemView().view(SEQUENCES_VIEW);
+
+            assertEquals(2, seqs0.size());
+            assertEquals(0, seqs1.size());
```

Comment:
> Is there a reason don't show the items in a view on other node before explicit touch them

System views just reflect the current behaviour.
I think it's OK to keep it as is, but I agree with you - it can confuse some users.
Will add a note about this detail in the documentation.

Reply:
> also by a reason that the backup value is 1 and the ds should already exist on g1

Ignite Data structures has 2 place that reflects their state:

1. system cache
2. `DataStructureProcessor#dsMap` (`CacheDataStructuresManager`)

I prefer to show second place, because, scanning a cache may involve network operations.
And network operations violate the local nature of the system views.

Reply:
Is it possible to touch this "seq-2" on the g1? I set create=false for "seq-1" on the g1 node, and it still works. But touching "seq-2" with create=false does nothing (not appeared in a view). Is it expected behaviour?

Reply:
`create=false` means, don't create if it not exists.
So value of this flag during `g1.atomicSequence("seq-1", 42, true);` don't affects test results.
Because, "seq-1" exists in the moment of `g1.atomicSequence("seq-1", 42, true);` invocation.